### ISSUE TITLE
Widget JS broken response on SaaS deployment

### ISSUE THREAD

**Issue by RaitoBezarius** [pretalx/pretalx#1562]
### Problem and impact

When trying to do `curl "https://talks.nixcon.org/nixcon-2023/schedule/widget/v2.en.js"`, this gets a broken partial transfer.

### Expected behaviour

Full transfer.

### Steps to reproduce

1. `curl "https://talks.nixcon.org/nixcon-2023/schedule/widget/v2.en.js"`

### Link

https://talks.nixcon.org/nixcon-2023/schedule/widget/v2.en.js

**Comment by mweinelt** [comment]
```
curl -o /tmp/foo https://talks.nixcon.org/nixcon-2023/schedule/widget/v2.en.js
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
 21  436k   21 97650    0     0   557k      0 --:--:-- --:--:-- --:--:--  560k
curl: (18) transfer closed with 348948 bytes remaining to read
```

**Comment by rixx** [comment]
Cannot reproduce, download proceeds normally to 100% for me from multiple locations.

(Please don't use the issues here to report problems with pretalx.com. The issue tracker is meant to track bugs in pretalx and proposed features.)

**Comment by RaitoBezarius** [comment]
@rixx Where can we report them? I retried from multiple servers and I cannot have the download complete.

**Comment by RaitoBezarius** [comment]
Just to be clear, we retried on multiple ASN:

AS12322, AS12876, AS8881, AS3320, AS24940, AS8365 without success.

**Comment by rixx** [comment]
> Where can we report them? I retried from multiple servers and I cannot have the download complete.

If pretalx.com doesn't work correctly, the best place would be the support form or the support email address (support@pretalx.com). 

I was able to reproduce the broken transmission on one out of seven servers/locations. I have no idea why/how it would be caused – I restarted the entire network stack on pretalx.com, and the problem seems to be gone, so I'd wager it was either something going wonky over very long uptimes, or Hetzner being Hetzner.
